GameWindow.java

```diff
@@ -662,160 +637,118 @@ private void render() // everything in the game that renders
 			return;
 		}
 
-		if (a == 1 && b == 1 && (c == 1 || c1 == 1 && c2 == 1 && c3 == 1)) {
-			score++;
+		if (a == 1 && b == 1 && (c1== 1&&c2== 1&&c3== 1||c ==1 )) {
+			
 			a = 0;
 			b = 0;
 			c = 0;
-			c1 = 0;
-			c2 = 0;
-			c3 = 0;
+			c1=0;
+			c2=0;
+			c3=0;
 		}
-
+		
 		Graphics g = bs.getDrawGraphics();
 		/////////////////////////////////////
 		g.drawImage(image, 0, 0, getWidth(), getHeight(), this);
-
-		if (state == STATE.START) {
-			// g.drawImage(title, 303, 200, this);
+		
+		if(state == STATE.START) 
+		{
+			g.drawImage(title,290,200,this);
 			g.setFont(new Font("Arial", Font.BOLD, 10));
 			g.setColor(Color.WHITE);
-			g.drawRect(258, 280, 125, 20);
-			g.drawString("PLAY", 305, 294);
-			g.setColor(Color.CYAN);
-			g.setFont(new Font("Arial", Font.BOLD, 70));
-			g.drawString("Melee Tech Trainer", 0, 200);
+			g.drawString("Press A" , 285, 280);
+		
 		}
-		if (state == STATE.BACKGROUNDS) {
-			g.setColor(Color.CYAN);
-			g.drawRect(0, 0, 50, 50);
-			g.setFont(new Font("Arial", Font.BOLD, 20));
-			g.drawString("ESC", 5, 32);
-			g.setFont(new Font("Arial", Font.BOLD, 70));
-			g.drawString("Choose a Stage", 55, 85);
-			g.setFont(new Font("Arial", Font.BOLD, 70));
-			g.drawRect(32, 130, 185, 70);
-			g.drawRect(32, 210, 185, 70);
-			g.drawRect(227, 130, 185, 70);
-			g.drawRect(227, 210, 185, 70);
-			g.drawRect(422, 130, 185, 70);
-			g.drawRect(422, 210, 185, 70);
-			g.setFont(new Font("Arial", Font.BOLD, 15));
-			g.setColor(Color.WHITE);
-			g.drawString("Final Destination", 64, 169);
-			g.drawString("Battlefield", 281, 169);
-			g.drawString("Dreamland", 474, 169);
-			g.drawString("Yoshi's Story", 76, 247);
-			g.drawString("Fountain of Dreams", 250, 247);
-			g.drawString("Pokemon Stadium", 450, 247);
-
+		
+	
+		
+		
+		
+	if(state == STATE.PLAY)	
+	{
+		g.setFont(new Font("Arial", Font.BOLD, 30));
+		g.setColor(Color.WHITE);
+		g.drawImage(controllerr, 200, 350, this);
+		g.drawImage(abutton, 327, 367, this);
+		g.drawImage(bbutton, 313, 372, this);
+		g.drawImage(xbutton, 339, 356, this);
+		g.drawImage(ybutton, 319, 353, this);
+		g.drawImage(lbutton, 238, 336, this);
+		g.drawImage(rbutton, 328, 336, this);
+		g.drawImage(startbutton, 282, 375, this);
+		g.drawImage(zbutton, 327, 345, this);
+		g.drawImage(stick, 244, 364, this);
+		g.drawImage(cstick, 307, 397, this); // UHJK

```

Comment:
Just like with the sounds, having the buttons in a map will probably be important both for your collision code and to make the code shorter and easier to understand.

Reply:
How can we add the buttons into a map when we use the "ss.grabImage()" to finish their instantiation. I wouldn't know what object type to use for the map.

Reply:
It would be a map from whatever you want to refer to them by (strings or enums, probably) to BufferedImages: the same type as the individual variables have now. It wouldn't shorten the initialization at all (it would just become `.put("name", ss.grabImage(value))` rather than `name = ss.grabImage(value)`) but it would help you refer to those buttons programmatically going forward should you need to do so.